src/pages/home/ReportScreen.tsx

```diff
@@ -391,6 +391,16 @@ function ReportScreen({
         Report.openReport(reportIDFromRoute, reportActionIDFromRoute);
     }, [reportIDFromRoute, reportActionIDFromRoute]);
 
+    useEffect(() => {
+        if (!report.reportID) {
+            return;
+        }
+
+        if (report?.errorFields?.notFound) {
+            Report.clearReportNotFoundErrors(report.reportID);
```

Comment:
This doesn't seem right to me. Why is this needed? @rezkiy37

Reply:
@arosiclair, please check the condition above. The app checks if a report has `reportID`, if yes it means that the `notFound` error is not relevant anymore.

Reply:
That's not necessarily true. If you lose access to a report,`report.errorFields.notFound` will be set along with the rest of the report data including the `reportID`.

So why is this needed? `errorFields.notFound` should already be cleared in the [successData for OpenReport](https://github.com/Expensify/App/blob/main/src/libs/actions/Report.ts#L786-L794).

Reply:
There was a bug - https://github.com/Expensify/App/pull/35671#issuecomment-2045481675. I fixed it - https://github.com/Expensify/App/pull/35671#issuecomment-2045352094.

Reply:
Okay it sounds like the `notFound` error is not cleared when we push reports to users. I think we can just fix that on the backend. Do you know which API command you were testing that caused that bug? cc @MonilBhavsar

Reply:
Looks like **InviteToRoom**

Reply:
Alright looks like we just need to update [Report::structureReportForOnyx](https://github.com/Expensify/Auth/blob/5110e614fe472804988dbb845536a3565e138016/auth/lib/Report.cpp#L8632) to clear `errorFields.notFound`. I opened an issue to clean this up [here](https://github.com/Expensify/App/issues/43357).